docs_user/assemblies/assembly_prepare_director_operator_and_rhoso_for_adoption_process.adoc

```diff
@@ -0,0 +1,190 @@
+ifdef::context[:parent-context: {context}]
+
+[id="prepare_director_operator_and_rhoso_for_adoption_process"]
+
+:context: migrating-databases
+
+= Preparing director operator for adoption
+
+[NOTE]
+====
+The example use case presented here is of a single OCP cluster, being shared by the source ODPdO env. _and_ a destination RHOSO env. Two OCP master nodes would be transferred to RHOSO use, leaving OSPdO with one for the adoption process, after which the remaining one can be used by RHOSO env. as well.
+====
+
+Work in the openstack project:
```

Comment:
```suggestion
. Log in to the {OpenShift } environment where {OpenStackShort} OSPdO is deployed and change to the project that hosts your {OpenStackShort} deployment:
```
I stole this text from [1]. I think this still applies.
[1] https://docs.redhat.com/en/documentation/red_hat_openstack_platform/16.2/html-single/service_telemetry_framework_1.5/index#deploying-the-overcloud-for-director-operator_assembly-completing-the-stf-configuration-using-director-operator

Reply:
ok